packages/@react-stately/color/src/useColorFieldState.ts

```diff
@@ -152,7 +152,7 @@ export function useColorFieldState(
   let incrementToMax = () => safelySetColorValue(MAX_COLOR);
   let decrementToMin = () => safelySetColorValue(MIN_COLOR);
 
-  let validate = (value: string) => value === '' || !!value.match(/^#?[0-9a-f]{0,6}$/i)?.[0];
+  let validate = (value: string) => value === '' || !!value.match(/^#?[0-9a-f]{0,8}$/i)?.[0];
```

Comment:
Should it always be enabled? I think there are a lot of cases where you wouldn't want users entering a value with an alpha component. Maybe it should be a prop?

Reply:
Fair, like any of these?
```
withAlpha?: boolean
fourChannel?: boolean
channels?: 'rgb' | 'rgba' -> default 'rgb'
format?: 'rgb' | 'rgba' -> default 'rgb'
```

Reply:
I think there are two use cases:

1. The ColorField is part of a larger color picker, which has a slider or something to set the alpha value. In this case, you might not want to display the alpha value in the ColorField, but it shouldn't be overwritten when setting the value. So, the color may have an alpha, but it wouldn't be touched by ColorField. I would say this should be the default behavior.
2. You want to allow the user to edit the alpha as part of the ColorField. In this case, if set to a 3 or 6 character value, the alpha should be overwritten to be 100%, but if set to a 4 or 6 character value, then it should set the alpha as parsed.

With that in mind, I guess I'd name it something like `allowsAlpha`, `allowsEditingAlpha`, or something. That would follow our naming guidelines.